apps/studio/package.json

```diff
@@ -124,6 +124,7 @@
     "ajv-errors": "^3.0.0",
     "date-fns": "^4.1.0",
     "date-fns-tz": "^3.1.3",
+    "dayjs": "^1.11.13",
```

Comment:
issue: I just saw that we are using `dayjs`. We should prefer to use `date-fns` instead cos `dayjs` configurations are set globally and not scoped to a specific execution, which can cause conflicts in other parts of the codebase.